swagger.yaml

```diff
@@ -2716,6 +2716,52 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /v2/operators/bundles:
+    get:
+      tags:
+        - operators
+      summary: Get list of avaliable bundles 
+      description: Retrieves a list of avaliable bundles.
+      operationId: V2ListBundles
+      responses:
+        "200":
+          description: Success
+          schema:
+            type: array
+            items:
+              type: string
```

Comment:
Should this return a list of `bundle` objects to make it more consistent with the `.../bundles/{bundle_name}` endpoint?